core/src/main/scala/package.scala

```diff
@@ -48,6 +48,7 @@ package object funnel {
     def append(a: A, b: => A) = G.plus(a, b)
   }
 
-  implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
-  implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
+  // This doesn't work in 2.11; these are at the highest priority within the package.
+  // implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
+  // implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
 }
```

Comment:
Ah ok, I see what you're doing. I think there are 4 cases where you're using the default `Strategy` that the 2.10 compiler just didn't catch.

```
[error] /Users/rnorris/Scala/funnel/core/src/main/scala/Monitoring.scala:278: ambiguous implicit values:
[error]  both value dontUseTheDefaultStrategy in package funnel of type => scalaz.concurrent.Strategy
[error]  and value theDefaultStrategyCausesProblems in package funnel of type => scalaz.concurrent.Strategy
[error]  match expected type scalaz.concurrent.Strategy
[error]     e(this).zip(alive.continuous).map(_._1).either(pts)
[error]                                                   ^
[error] /Users/rnorris/Scala/funnel/core/src/main/scala/Monitoring.scala:297: ambiguous implicit values:
[error]  both value dontUseTheDefaultStrategy in package funnel of type => scalaz.concurrent.Strategy
[error]  and value theDefaultStrategyCausesProblems in package funnel of type => scalaz.concurrent.Strategy
[error]  match expected type scalaz.concurrent.Strategy
[error]     e(this).zip(alive.continuous).map(_._1).either(pts)
[error]                                                   ^
[error] /Users/rnorris/Scala/funnel/core/src/main/scala/Monitoring.scala:393: ambiguous implicit values:
[error]  both value dontUseTheDefaultStrategy in package funnel of type => scalaz.concurrent.Strategy
[error]  and value theDefaultStrategyCausesProblems in package funnel of type => scalaz.concurrent.Strategy
[error]  match expected type scalaz.concurrent.Strategy
[error]   val costiveKeys: Signal[Set[Key[Any]]] = signalOf(Set())
[error]                                                    ^
[error] /Users/rnorris/Scala/funnel/core/src/main/scala/Monitoring.scala:510: ambiguous implicit values:
[error]  both value dontUseTheDefaultStrategy in package funnel of type => scalaz.concurrent.Strategy
[error]  and value theDefaultStrategyCausesProblems in package funnel of type => scalaz.concurrent.Strategy
[error]  match expected type scalaz.concurrent.Strategy
[error]      scalaz.stream.merge.mergeN(M.distinctKeys.filter(p).map(k => points(k)))
[error]                                ^
```